tests/Unit/CssInlinerTest.php

```diff
@@ -2252,6 +2252,20 @@ public function inlineCssAppliesCssToMatchingElementsAndKeepsRuleWithPseudoCompo
         self::assertContainsCss('<style type="text/css">a:hover { color: green; }</style>', $result);
     }
 
+    /**
+     * #844
```

Comment:
To keep the code succinct, please let's not have issue numbers in the code. Having them in the changelog and the git history should suffice.